controllers/operator/clusterpodplacementconfig_controller_test.go

```diff
@@ -580,12 +582,101 @@ var _ = Describe("Controllers/ClusterPodPlacementConfig/ClusterPodPlacementConfi
 				validateReconcile()
 			})
 		})
-		AfterAll(func() {
-			err := k8sClient.Delete(ctx, builder.NewClusterPodPlacementConfig().WithName(common.SingletonResourceObjectName).Build())
-			Expect(err).NotTo(HaveOccurred(), "failed to delete ClusterPodPlacementConfig", err)
-			Eventually(framework.ValidateDeletion(k8sClient, ctx)).Should(Succeed(), "the ClusterPodPlacementConfig should be deleted")
+		Context("When a v1beta1 pod placement config is created", func() {
+			AfterEach(func() {
+				By("Deleting the ClusterPodPlacementConfig")
+				err := k8sClient.Delete(ctx, builder.NewClusterPodPlacementConfig().WithName(common.SingletonResourceObjectName).Build())
+				Expect(err).NotTo(HaveOccurred(), "failed to delete ClusterPodPlacementConfig", err)
+				Eventually(framework.ValidateDeletion(k8sClient, ctx)).Should(Succeed(), "the ClusterPodPlacementConfig should be deleted")
+			})
+			It("should be possible to get the CPPC as a v1alpha1 with no specs regarding the plugins", func() {
+				By("Creating a v1beta1 ClusterPodPlacementConfig")
+				// err := k8sClient.Create(ctx, builder.NewClusterPodPlacementConfig().WithName(common.SingletonResourceObjectName).Build())
+				// Expect(err).NotTo(HaveOccurred(), "failed to create ClusterPodPlacementConfig", err)
+				// validateReconcile()
+				// Get the details
+				ppc := &v1beta1.ClusterPodPlacementConfig{}
+				err := k8sClient.Get(ctx, crclient.ObjectKeyFromObject(&v1beta1.ClusterPodPlacementConfig{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      common.SingletonResourceObjectName,
+						Namespace: utils.Namespace(),
+					},
+					Spec: v1beta1.ClusterPodPlacementConfigSpec{
+						LogVerbosity: "Normal",
+						Plugins: plugins.Plugins{
+							NodeAffinityScoring: &plugins.NodeAffinityScoring{
+								BasePlugin: plugins.BasePlugin{
+									Enabled: true,
+								},
+								Platforms: []plugins.NodeAffinityScoringPlatformTerm{
+									{Architecture: utils.ArchitectureArm64, Weight: 50},
+								},
+							},
+						},
+					},
+				}), ppc)
+				Expect(err).NotTo(HaveOccurred())
+				Eventually(func(g Gomega) {
+					// Verify the LogVerbosity field
+					g.Expect(ppc.Spec.LogVerbosity).NotTo(Equal("Normal"))
+					// Verify the Plugins field
+					g.Expect(ppc.Spec.Plugins.NodeAffinityScoring).To(BeNil())
+				}).Should(Succeed())
+			})
+			It("does not query the remote registry until the cache expires", func() {
+				cppc := &v1beta1.ClusterPodPlacementConfig{ObjectMeta: metav1.ObjectMeta{
```

Comment:
We should use the builder here.

```suggestion
				err := k8sClient.Create(ctx, builder.NewClusterPodPlacementConfig().WithName(common.SingletonResourceObjectName).WithNodeAffinityScoring(true).WithNodeAffinityScoringTerm(utils.ArchitectureArm64, 50).Build())
				Expect(err).NotTo(HaveOccurred(), "failed to create ClusterPodPlacementConfig", err)
				validateReconcile()
```

Where WithNodeAffinityScoringPlugin and WithNodeAffinityScoringTerm are two methods to be implemented in https://github.com/openshift/multiarch-tuning-operator/blob/main/pkg/testing/builder/clusterpodplacementconfig.go.

The first only gets the value for the `enabled` field, the second gets (string, int) to append an element in the Platforms slice of the Plugins.NodeAffinityScoring spec of the CPPC.